index.html

```diff
@@ -1,19 +1,115 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <header class="header">
+        <img src="file:///C:/Users/New%20User/Documents/GitHub/HTML-CSS-Coursework-Week1/HTML-CSS-Coursework-Week2/favicon.ico" alt="logo">
+        <nav class="nav-bar">
+            <ul>
+                <a>meet karma</a>
+                <a>how its works</a>
+                <div id="red"><a>store</a></div>
+                <a>blog</a>
+                <a>help</a>
+                <a>login</a>
```

Comment:
```suggestion
                <li><a>meet karma</a></li>
                <li><a>how its works</a></li>
                <li id="red"><a>store</a></li>
                <li><a>blog</a></li>
                <li><a>help</a></li>
                <li><a>login</a></li>
```
These links are within a `ul` element, so we should put them in `li` ones